WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -466,8 +451,7 @@ private extension ProductsViewController {
             self?.updateTableHeaderViewHeight()
         }, onGiveFeedbackButtonPressed: { [weak self] in
             self?.presentProductsFeedback()
-        }, onDismissButtonPressed: { [weak self] in
-            self?.dismissProductsBanner()
```

Comment:
Since we're leaving the code to present this banner, how about replacing this with a method that still dismisses the banner? We could use `hideTopBannerView()` to dismiss the banner without making any calls to the `updateFeedbackStatus` action.

Reply:
Thanks for the pointer, I didn't want to create a new dismiss function to not use it, but I didn't see we can use the existing `hideTopBannerView()` for this.